python/paddle/v2/fluid/layers/nn.py

```diff
@@ -1504,3 +1504,54 @@ def reduce_min(input, dim=None, keep_dim=False):
             'reduce_all': True if dim == None else False
         })
     return out
+
+
+def warpctc(input, label, blank=0, norm_by_times=False, **kwargs):
+    """
+    An operator integrating the open source warp-ctc  library
```

Comment:
1. warp-ctc  library -> Warp-CTC library, and URL.

Reply:
Thx. Done.